README.md

```diff
@@ -33,19 +33,19 @@ Read more at
 Codecs supported for CD Digital Audio emulation (loading CD music via
 [cue sheets](https://en.wikipedia.org/wiki/Cue_sheet_(computing))):
 
-|                | dosbox-staging<sup>🞰</sup> | DOSBox<sup>†</sup>
+|                | dosbox-staging<sup>†</sup> | DOSBox<sup>‡</sup>
 |-               |-                           |-
 | **Opus**       | Yes (libopus)              | No
-| **OGG/Vorbis** | Yes (built-in)             | Yes - SDL\_sound 1.2 (libvorbis)<sup>[6],‡</sup>
-| **MP3**        | Yes (built-in)             | Yes - SDL\_sound 1.2 (libmpg123)<sup>[6],‡,§</sup>
+| **OGG/Vorbis** | Yes (built-in)             | Yes - SDL\_sound 1.2 (libvorbis)<sup>[6],＊</sup>
+| **MP3**        | Yes (built-in)             | Yes - SDL\_sound 1.2 (libmpg123)<sup>[6],＊,§</sup>
 | **FLAC**       | Yes (built-in)             | No<sup>§</sup>
-| **WAV**        | Yes (built-in)             | Yes - SDL\_sound 1.2 (internal)<sup>[7],‡</sup>
-| **AIFF**       | No                         | Yes - SDL\_sound 1.2 (internal)<sup>[7],‡</sup>
+| **WAV**        | Yes (built-in)             | Yes - SDL\_sound 1.2 (internal)<sup>[7],＊</sup>
+| **AIFF**       | No                         | Yes - SDL\_sound 1.2 (internal)<sup>[7],＊</sup>
```

Comment:
Didn't know those existed; very nice!

Reply:
The wiki article about "Asterisk" lists lots and lots of them :)